website/content/docs/job-specification/connect.mdx

```diff
@@ -9,12 +9,20 @@ description: |-
 
 <Placement groups={['job', 'group', 'service', 'connect']} />
 
-The `connect` block allows configuring various options for
-[Consul Connect](/nomad/docs/networking/consul). It is
-valid only within the context of a service definition at the task group
-level. For using `connect` when Consul ACLs are enabled, be sure to read through
-the [Secure Nomad Jobs with Consul Connect](/nomad/tutorials/integrate-consul/consul-service-mesh)
-guide.
+Use the `connect` block to configure various options for Consul service mesh. It
+is valid only within the context of a service definition at the task group
+level.
```

Comment:
Because we're stuck calling this `connect`, even though Consul's branding has changed, should we include a "formerly known as" remark at least once in the docs? (This seems like as good a spot as any)